samples/snippets/visualbasic/VS_Snippets_Misc/tpl_synccontext/vb/MainWindow.xaml.vb

```diff
@@ -48,7 +48,7 @@ Partial Public Class MainWindow : Inherits Window
             images(x) = Task.Factory.StartNew(Function() LoadImage(files(x)))
         Next
 
-        ' When they�ve all been loaded, tile them into a single byte array.
+        ' When they have all been loaded, tile them into a single byte array.
```

Comment:
Tiny nitpick: why does C# use "they've", but VB uses "they have"?

Reply:
I just try and avoid the ' character in VB comments. I know it's not strictly necessary, but it just looks bad to my eyes.  🤷‍♂️